src/javascript/JContent/JContent.constants.js

```diff
@@ -24,6 +26,13 @@ const JContentConstants = {
     },
     localStorageKeys: {
         filesSelectorMode: 'jcontent_files_selector_mode'
+    },
+    accordionPermissions: {
+        pagesAccordionAccess: 'pagesAccordionAccess',
+        contentFolderAccordionAccess: 'contentFolderAccordionAccess',
+        mediaAccordionAccess: 'mediaAccordionAccess',
+        additionalAccordionAccess: 'additionalAccordionAccess',
+        formAccordionAccess: 'formAccordionAccess'
```

Comment:
maybe we don't need this one since it's been moved to forms?

Reply:
We still need this as on the "jContent button" button click action is still in jContent repository (generic code)
We need to check here for the form access permissions.

Reply:
ok good to know, thanks